File: nivel01/respostas/c/exercicio05.c
#include <stdio.h>

int main(){
	
	float totalEleitores, numVotosBrancos, numVotosNulos, numVotosValidos;
	float porcentagemVotosBrancos, porcentagemVotosNulos, porcentagemVotosValidos;

	printf("Digite o numero total de eleitores: ");
	scanf("%f", &totalEleitores);

	printf("Digite o numero total de votos em branco: ");
	scanf("%f", &numVotosBrancos);

	printf("Digite o numero total de votos nulos: ");
	scanf("%f", &numVotosNulos);

	printf("Digite o numero total de votos validos: ");
	scanf("%f", &numVotosValidos);

	porcentagemVotosBrancos = (numVotosBrancos / totalEleitores) * 100;
	porcentagemVotosNulos = (numVotosNulos / totalEleitores) * 100;
	porcentagemVotosValidos = (numVotosValidos / totalEleitores) * 100;

	printf("Porcentagem de votos brancos: %0.f", porcentagemVotosBrancos);
	printf("\n");
	printf("Porcentagem de votos nulos: %0.f", porcentagemVotosNulos);
	printf("\n");
	printf("Porcentagem de votos validos: %0.f", porcentagemVotosValidos);
	printf("\n");

	return 0;

}

File: nivel01/respostas/ruby/exercicio05.rb
print "Numero de votos dos eleitores: "
eleitores = gets.to_i
print "Numero de votos em branco do total de votos do eleitores: "
brancos = gets.to_i
print "Numero de votos nulos do total de votos do eleitores: "
nulos = gets.to_i
validos = eleitores - (brancos + nulos)
puts "Numero de votos validos: #{validos} "


total_brancos =(brancos * 100) / eleitores
total_nulos = (nulos * 100) / eleitores
total_validos = (validos * 100)/ eleitores

puts "#{total_brancos}% foram votos brancos dos eleitores"
puts "#{total_nulos}% foram votos nulos dos eleitores"
puts "#{total_validos}% foram votos validos do eleitores"
